src/prop/theory_proxy.cpp

```diff
@@ -345,6 +345,15 @@ std::vector<Node> TheoryProxy::getLearnedZeroLevelLiterals(
   return {};
 }
 
+modes::LearnedLitType TheoryProxy::getLiteralType(const Node& lit) const
+{
+  if (d_zll != nullptr)
+  {
+    return d_zll->computeLearnedLiteralType(lit);
+  }
+  return {};
```

Comment:
Does this type check? Maybe we can introduce an UNKNOWN to the enum LearnedLitType?

Reply:
Ah, this is a copy/paste artifact. 

An UNKNOWN LearnedLitType would be a good idea. Currently there is a default LearnedLitType that this returns even if produce-learned-literals is not enabled (I forget which one, but it does not make sense).

As for returning an empty vector, I'm surprised that there have not been any compiler errors/warnings. I'll fix this, though.